updater/updater.py

```diff
@@ -30,14 +30,16 @@
 
 [Service]
 Type=simple
-ExecStart={0} --log-file /var/log/lepidopter-update.log update --watch
+Environment="UPDATER=/opt/ooni/lepidopter-update/updater/updater.py"
+ExecStart=$UPDATER --log-file /var/log/ooni/lepidopter-update.log update --watch
 TimeoutStartSec=300
 Restart=on-failure
+
 [Install]
 WantedBy=multi-user.target
 """.format(SCRIPT_INSTALL_PATH)
```

Comment:
If you remove the `{0}` you should also remove the `.format()` call at the end. I would suggest you keep it as that performs substitution of the variable `{0}` with the actual path to the script